master/.golangci.yml

```diff
@@ -85,3 +88,38 @@ linters:
     - testpackage
     - unparam
     - wsl
+
+    # Below here are new linters from upgrading to 1.43.0. Since we enable all and disable
+    # selectively, when we upgrade we get a ton of new linters. For convenience next upgrade,
+    # golangci-lint can tell you which linters are enabled:
+    #   golangci-lint linters | sed -n '/Enabled/,/Disabled/p'
+    # Only in the new list were disabled with this change:
```

Comment:
Don't understand this line. Missing something from the comment?

Reply:
tried to make it clearer